entrypoint.sh

```diff
@@ -107,27 +93,17 @@ restore() {
 list-backups() {
     local batch_mode=$([[ "$1" = "--batch-mode" ]] && echo true || echo false)
     local files_in_s3=();
-    if is_aws_s3_enabled; then
+    if s3sync is_enabled; then
```

Comment:
Maybe an `else` here as well?  Or maybe instead of all the else statements the `is_enabled` function could print the message to say it's not enabled.